hack/versions/pkg/schema/git.go

```diff
@@ -73,7 +74,7 @@ func (g *git) diffWithBaseline(path string) ([]byte, error) {
 
 func (g *git) run(args ...string) ([]byte, error) {
 	cmd := exec.Command(g.path, args...)
-	out, err := util.RunCmdOut(cmd)
+	out, err := util.RunCmdOut(context.Background(), cmd)
```

Comment:
There is another PR to move replace context.Background to context.TODO.
Do you want to sync with that and see why `TODO` is preferred?

From docs, looks like Background context is never cancelled. 
https://pkg.go.dev/context#Background

Reply:
I think that `context.TODO` is a hint to the dev that if ever a proper context becomes available in future by a change in the function parameters or something, then plumb that through. Whereas `context.Background` is a deliberate use even if a parent context is available. That's what I understood from @ahmetb's PR.

Reply:
yeah that's the case. ideally a context.Background() is for a call stack that can be cancelled from a central place, e.g. Ctrl-C. Background threads with separate lifecycle (as the name also says) get a new Background context.

If you don't plumb yet but intend do plumb someday, you can use context.TODO, which is identical to Background() functionally, only different in semantics and indicates your intention.